tabulate/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "0.7.5" %}
+
+package:
+  name: tabulate
+  version: {{ version }}
+
+source:
+  fn: tabulate-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/t/tabulate/tabulate-{{ version }}.tar.gz
+  md5: 576e1f063b8e74dbfeda02d978564987
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
Will try to use `setuptools` if it can find it and sometimes `conda` throws it in anyways. So, figured we just use it explicitly.